### PR TITLE
feat: add ToUppercase class

### PR DESCRIPTION
**Pull request by RogerMo01**
This problem can also be resolved in a non-recursive way in just one line:

return word.ToUpper();

### CODE REVIEW

File: Recursive Exercices/Recursive Exercices/ToUppercase.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+namespace Recursive_Exercices
+{
+    class ToUppercase
+    {
+        public static string ToUpperC(string word)
+        {
+            return Convert(word, "");
+        }
+
+        private static string Convert(string word, string returnWord)
```
Comment (LongJohnSilver1504) [inline Recursive Exercices/Recursive Exercices/ToUppercase.cs:16]: Instead f returnWorld use result

File: Recursive Exercices/Recursive Exercices/ToUppercase.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        {
+            if (returnWord.Count() == word.Count())
```
Comment (LongJohnSilver1504) [inline Recursive Exercices/Recursive Exercices/ToUppercase.cs:18]: Use ternary expression

File: Recursive Exercices/Recursive Exercices/ToUppercase.cs (hunk 1 above)
Comment (RogerMo01) [reply Recursive Exercices/Recursive Exercices/ToUppercase.cs:16]: Done!

File: Recursive Exercices/Recursive Exercices/ToUppercase.cs (hunk 2 above)
Comment (RogerMo01) [reply Recursive Exercices/Recursive Exercices/ToUppercase.cs]: Done!

**Pull request merged**